test/test.js

```diff
@@ -0,0 +1,23 @@
+var Lab = require('lab');
+var Code = require('code');
+
+var lab = exports.lab = Lab.script();
+var describe = lab.describe;
+var it = lab.it;
+var expect = Code.expect;
+
+var Fs = require('fs');
+var Htmlterminal = require('..');
```

Comment:
Please use lowercase here, you can call it just `hget` since that's what you're requiring